packages/legend-application-studio/src/components/__tests__/EmbeddedQueryBuilder.test.tsx

```diff
@@ -253,6 +429,175 @@ test(integrationTest('Open query builder by querying a class'), async () => {
   );
 });
 
+test(
```

Comment:
Let's separate these tests to another file, since these focus one particular workflow, feel free to name it something like `ClassQueryBuilder.test.tsx`, the test data can be moved to its own file. Also, these tests can be cleaned up to use `TestCase` I believe - that's what I meant by making these tests scale better